package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "datamegh",
+  "version": "1.0.0-alpha.1",
+  "description": "Datamegh - Data Engineering for the cloud.",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
```suggestion
    "typecheck": "pyright"
```

Reply:
And add this in the make file such that when you run `make build` or `make test` - this type checking is run first before any other steps.